docs/plans/reviews/2026-04-22-issue-29-antipatterns.md

```
@@ -0,0 +1,97 @@
+# Issue #29 Test Plan — Antipattern Review
+
+Canon: codepipes 13 + modern (snapshot abuse, AI-generated tests, ice cream cone, brittle selectors, over-mocking, missing contracts, assertion-free tests).
+
+## 1. Antipattern findings
+
+### Task 1 — `empty_project_returns_empty_vec`
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)